lib/studies/reply/adaptive_nav.dart

```diff
@@ -415,68 +370,87 @@ class _NavigationRailFolderSection extends StatelessWidget {
-            ),
-            child: Text(
-              'FOLDERS',
-              style: textTheme.caption.copyWith(
-                color: navigationRailTheme.unselectedLabelTextStyle.color,
-              ),
-            ),
-          ),
-          const SizedBox(height: 8),
-          for (var folder in folders.keys)
-            InkWell(
-              borderRadius: const BorderRadius.all(
-                Radius.circular(36),
-              ),
-              onTap: () {},
-              child: Column(
-                children: [
-                  Row(
-                    children: [
-                      const SizedBox(width: 12),
-                      ImageIcon(
-                        AssetImage(
-                          folders[folder],
-                          package: _assetsPackage,
-                        ),
-                        color:
-                            navigationRailTheme.unselectedLabelTextStyle.color,
+    final animation = NavigationRail.extendedAnimation(context);
+
+    return AnimatedBuilder(
+      animation: animation,
+      builder: (context, child) {
+        return Visibility(
+          maintainAnimation: true,
+          maintainState: true,
+          visible: animation.value > 0,
+          child: Opacity(
+            opacity: animation.value,
+            child: Align(
+              widthFactor: animation.value,
+              alignment: AlignmentDirectional.centerStart,
+              child: SizedBox(
+                height: 485,
+                width: 256,
+                child: ListView(
+                  padding: const EdgeInsets.all(12),
+                  physics: const NeverScrollableScrollPhysics(),
+                  children: [
+                    const Divider(
+                      color: ReplyColors.blue200,
+                      thickness: 0.4,
+                      indent: 14,
+                      endIndent: 16,
+                    ),
+                    const SizedBox(height: 16),
+                    Padding(
+                      padding: const EdgeInsetsDirectional.only(
+                        start: 16,
                       ),
-                      const SizedBox(width: 24),
-                      Text(
-                        folder,
-                        style: textTheme.bodyText1.copyWith(
+                      child: Text(
+                        'FOLDERS',
+                        style: textTheme.caption.copyWith(
                           color: navigationRailTheme
                               .unselectedLabelTextStyle.color,
                         ),
                       ),
-                      const SizedBox(height: 72),
-                    ],
-                  ),
-                ],
+                    ),
+                    const SizedBox(height: 8),
+                    for (var folder in folders.keys)
+                      InkWell(
+                        borderRadius: const BorderRadius.all(
```

Comment:
nit: can `borderRadius` fit on one line?

Reply:
It does not fit